FHSDK/FH.cs

```diff
@@ -319,6 +321,11 @@ public static FHAuthSession GetAuthSession()
 			return headers;
 		}
 
+        public static async void Push(EventHandler<PushReceivedEvent> HandleNotification)
```

Comment:
Might be better call it PushRegister to keep it consistant with ios and android sdks.